### ISSUE TITLE
The demo url of posrgres from goctl tools usage is not correct.

### ISSUE THREAD

**Issue by ak5w** [zeromicro/go-zero#4029]
**Describe the bug**
The **demo url of posrgres** from goctl tools usage is not correct. like this
```
The data source of database,like "root:password@tcp(127.0.0.1:3306)/database
```
seems copy from mysql side.

But the usage info from go-zero doc site is correct, 
[goctl model](https://go-zero.dev/docs/tutorials/cli/model#postgresql-%E7%B1%BB%E5%9E%8B%E6%98%A0%E5%B0%84%E5%85%B3%E7%B3%BB)

**To Reproduce**

1. env
    GOCTL_VERSION=1.6.3
2. using below command
    goctl model pg datasource -h

**Expected behavior**
   should provider a correct and friendly url info in usage, it is important for a beginner.

**Comment by ak5w** [comment]
I raised a PR for fixing this.
[fix the usage datasource url of postgresql (#4029)](https://github.com/zeromicro/go-zero/pull/4030)

**Comment by kevwan** [comment]
fixed by #4030
